docs/user/tutorials/01-50-expose-and-secure-a-workload/01-52-expose-and-secure-workload-jwt.md

```diff
@@ -4,30 +4,56 @@ This tutorial shows how to expose and secure Services using APIGateway Controlle
 
 ## Prerequisites
 
-* Deploy a [sample HTTPBin Service](../01-00-create-workload.md).
-* [JSON Web Token (JWT)](./01-51-get-jwt.md)
-* Set up [your custom domain](../01-10-setup-custom-domain-for-workload.md) or use a Kyma domain instead. 
-* Depending on whether you use your custom domain or a Kyma domain, export the necessary values as environment variables:
-  
-  <!-- tabs:start -->
-  #### **Custom Domain**
-      
-  ```bash
-  export DOMAIN_TO_EXPOSE_WORKLOADS={DOMAIN_NAME}
-  export GATEWAY=$NAMESPACE/httpbin-gateway
-  ```
-  #### **Kyma Domain**
+* [Deploy a sample HTTPBin Service](../01-00-create-workload.md).
+* [Obtain a JSON Web Token (JWT)](./01-51-get-jwt.md).
+* [Set up your custom domain](../01-10-setup-custom-domain-for-workload.md) or use a Kyma domain instead.
+
+## Steps
+
+## Expose and Secure Your Workload
```

Comment:
Is it correct that this is on the same header level as Steps?
```suggestion
### Expose and Secure Your Workload
```

Reply:
It's not, thank for spotting this